### PR TITLE
Fix alpha bugs for relevant features

### PR DESCRIPTION
**Pull request by starbucksventi**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Birthday.java [hunk 1]
```diff
@@ -12,13 +12,12 @@
...
 
-    public static final String MESSAGE_CONSTRAINTS = "Birthday should be a valid date of the format yyyy-mm-dd and be"
-            + " within reasonable limits.";
+    public static final String MESSAGE_INVALID_BIRTHDAY_AFTER_PRESENT = "Birthday cannot be in the future.";
     public static final String BIRTHDAY_REMINDER_HEADER = "Upcoming birthdays: \n";
     public static final String BIRTHDAY_REMINDER_EMPTY = "No upcoming birthdays.\n";
     public static final String CUSTOM_BIRTHDAY_FORMAT = "'s birthday is on "; //Used for displaying a person's birthday
     public static final Birthday EMPTY_BIRTHDAY = Birthday.of("");
-    public static final LocalDate LOWER_BOUND = LocalDate.parse("1908-05-23");
+    public static final int FAILED_BIRTHDAY_PARSE_INDEX = 0;
```
Comment (minhao23) [inline src/main/java/seedu/address/model/person/Birthday.java]: What is the purpose of this line?

File: src/main/java/seedu/address/logic/parser/BirthdayCommandParser.java [hunk 2]
```diff
@@ -26,9 +29,13 @@ public BirthdayCommand parse(String args) throws ParseException {
             index = ParserUtil.parseIndex(argMultimap.getPreamble());
             String birthday = argMultimap.getValue(PREFIX_BIRTHDAY).orElse("");
             return new BirthdayCommand(index, new Birthday(birthday));
-        } catch (IllegalValueException | IllegalArgumentException ive_or_iae) {
+        } catch (IllegalValueException ive) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, BirthdayCommand.MESSAGE_USAGE),
-                    ive_or_iae);
+                    ive);
```
Comment (minhao23) [inline src/main/java/seedu/address/logic/parser/BirthdayCommandParser.java:34]: Is this for reformatting the new error message?
If it is then LGTM bro.

**Review by minhao23 — COMMENTED** [review]
Overall LGTM, just a few clarifications

File: src/main/java/seedu/address/logic/parser/BirthdayCommandParser.java (hunk 2 above)
Comment (starbucksventi) [reply src/main/java/seedu/address/logic/parser/BirthdayCommandParser.java:34]: Yes it is, one new one for wrong date format and the other for invalid birthdays in the future

**Pull request merged**